tidymodel-prediction-workflows-inside-snowflake-with-orbital/tidymodel-prediction-workflows-inside-snowflake-with-orbital.md

```diff
@@ -0,0 +1,1145 @@
+
+```r
+lendingclub_fit <- 
+  lendingclub_wf |> 
+  fit(data = lendingclub_prep)
+```
+
+To understand how well our model performs, we define a metric set using `yardstick::metric_set()`
+and then compute those metrics for `lendingclub_fit`.
+
+```r
+lendingclub_metric_set <- metric_set(rmse, mae, rsq)
+
+lendingclub_metrics <- 
+  lendingclub_fit |>
+  augment(lendingclub_prep) |>
+  lendingclub_metric_set(truth = INT_RATE, estimate = .pred)
+```
+
+`lendingclub_metrics` looks like this:
+
+```
+# A tibble: 3 × 3
+  .metric .estimator .estimate
+  <chr>   <chr>          <dbl>
+1 rmse    standard       4.38 
+2 mae     standard       3.43 
+3 rsq     standard       0.233
+```
+
+### Version model with vetiver
+
+The [vetiver](https://rstudio.github.io/vetiver-r/) package provides tools to version, share, 
+deploy, and monitor models. 
+
+We'll use vetiver to version and write our fitted model to a [pin](https://pins.rstudio.com/) on [Posit Connect](https://posit.co/products/enterprise/connect/). 
+This will allow us to identify which version of our model is active and track
+performance against other models over time. 
+
+First, we need to connect to a Posit Connect board. 
+
+```r
+board <- board_connect()
+```
+
+> aside negative
+>
+> To run `rsconnect::board_connect()`, you'll first need to authenticate. To authenticate,
+> navigate to `Tools` > `Global Options` > `Publishing` > `Connect` and follow the instructions.
+
+Then, we create a vetiver model with `vetiver_model()`, supplying the function with our 
+fitted model, model name, and metadata containing our metrics. 
+
+```r
+model_name <- "interest_rate_prediction"
+v <- 
+  vetiver_model(
+    lendingclub_fit,
+    model_name,
+    metadata = list(metrics = lendingclub_metrics)
+  )
+```
+
+`vetiver_pin_write()` writes the vetiver model to Posit Connect as a pin.
+
+```r
+board |> vetiver_pin_write(v)
+```
+
+> aside negative
+>
+> If you run into a namespacing error in Connect, or a permissioning error in Snowflake, that 
+> may mean someone has already run this code with the same `model_name`. You'll need to pick 
+> a different value. 
+
+We can use `pin_versions()` to return all the different versions of this model. 
+
+```r
+model_versions <- 
+  board |>
+  pin_versions(glue("{board$account}/{model_name}"))
+```
+
+> aside positive
+>
+> Unless you've done this Quickstart multiple times, you'll probably only have 
+> one model version.
+
+Let's grab the active version of the model. We'll use this later when interacting
+with Snowflake. 
+
+```r
+model_version <- 
+  model_versions |>
+  filter(active) |>
+  pull(version)
+```
+
+At this point, we've:
+
+1. Fit a model
+2. Evaluated that model
+3. Versioned the model with vetiver
+
+Now, we're ready to deploy the model with orbital and Snowflake. 
+
+## Predict with orbital and Snowflake
+Duration: 10
+
+The orbital package allows you to run tidymodels workflow predictions inside 
```

Comment:
```suggestion
The [orbital package](https://orbital.tidymodels.org/) allows you to run tidymodels workflow predictions inside 
```
I know we provide this link earlier in the document and that may be the best practice to follow here. It just feels like it may be worth linking back to orbital documentation again since orbital is being formally introduced into the workflow.